coderd/database/migrations/000232_update_dormancy_notification_template.down.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE notification_templates
+SET
+    body_template = E'Hi {{.UserName}}\n\n' || E'Your workspace **{{.Labels.name}}** has been marked as **dormant**.\n' || E'The specified reason was "**{{.Labels.reason}} (initiated by: {{ .Labels.initiator }})**\n\n' || E'Dormancy refers to a workspace being unused for a defined length of time, and after it exceeds {{.Labels.dormancyHours}} hours of dormancy might be deleted.\n' || E'To activate your workspace again, simply use it as normal.'
```

Comment:
Neither `initiator` nor `dormancyHours` labels are present, AFAICS.
Please test these notifications before submitting your PR.

Reply:
This is a "rollback" migration, so it should use the previous values from the applied migration. Maybe you got confused, thinking this was the "up" migration?

Reply:
I didn't notice that, you're correct - however I'm not really seeing the point of reverting back to the previous template?

Reply:
Me neither, but I suspect it could create some issues while changing branches or navigating commits since the previous code might not expect the new labels, for example. So, I decided to be extra safe.

Reply:
I don't think we want to go out-of-sync with the code calling these templates, so let's clear out the down migration here